### PR TITLE
Resolved Python 2 to Python 3 migration issue (#47)

### PR DESCRIPTION
**Pull request by Teerath-Agarwal**
This pull request addresses the issue described in #47, which involves migrating the codebase from Python 2 to Python 3. 

Changes made in this pull request:
- Updated the codebase to use Python 3.x syntax and features.
- Updated and verified that all dependencies are compatible with Python 3.
- Removed unnecessary code and dependencies.
- Tested the application with Python 3.10.11 to ensure compatibility.

### CODE REVIEW

File: erp.py [hunk 1]
```diff
@@ -90,17 +94,17 @@ def wrapped_func(*args, **kwargs):
                    **req_args)
 
         if len(r.history) < 2:
-            print("{answer} (ERP_A{index}) is wrong for {question}".format(
+            print(("{answer} (ERP_A{index}) is wrong for {question}".format(
```
Comment (harshkhandeparkar) [inline erp.py:97]: Is this bracket required?

File: migration/export_database.py [hunk 2]
```diff
@@ -29,13 +27,13 @@ def export_db():
     repeated_notices = []
     for notice in notices_cursor:
         try:
-            mc_new.get_default_database().notices.insert(notice)
-            print("inserted notice: ", notice)
+            mc_new.get_default_database().notices.insert_one(notice)
+            print(("inserted notice: ", notice))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:31]: Is this bracket required?

File: migration/export_database.py [hunk 3]
```diff
@@ -29,13 +27,13 @@ def export_db():
...
         except errors.DuplicateKeyError:
             print("entry already in database")
             repeated_notices.append(notice)
         except Exception as ex:
-            print("error in inserting: ", ex)
+            print(("error in inserting: ", ex))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:36]: This one too

File: migration/export_database.py [hunk 4]
```diff
@@ -65,8 +63,8 @@ def insert_notice(notice, mc_new):
         Returns 1 if some other error occurs
     '''
     try:
-        mc_new.get_default_database().notices.insert(notice)
-        print("inserted specific notice: ", notice)
+        mc_new.get_default_database().notices.insert_one(notice)
+        print(("inserted specific notice: ", notice))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:67]: This one as well

File: migration/export_database.py (hunk 2 above)
Comment (harshkhandeparkar) [reply migration/export_database.py:31]: Seems unnecessary and it may change the output.
image

File: migration/export_database.py [hunk 5]
```diff
@@ -75,7 +73,7 @@ def insert_notice(notice, mc_new):
         print("Error while connecting to DB")
         return 1
     except Exception as ex:
-        print("error in specific inserting: ", ex)
+        print(("error in specific inserting: ", ex))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:76]: This one as well

File: migration/export_database.py [hunk 6]
```diff
@@ -95,34 +93,26 @@ def insert_from_file(filename):
     further_repeated = []
     with open(filename, "r") as f:
         notices = loads(f.read())
-        print("Notice count: {}".format(len(notices)))
+        print(("Notice count: {}".format(len(notices))))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:96]: This one as well

File: migration/export_database.py [hunk 7]
```diff
@@ -95,34 +93,26 @@ def insert_from_file(filename):
...
         for notice in notices:
             out = insert_notice(notice, mc_new)
             if(out == 1):
                 further_defaulters.append(notice)
             elif(out == 2):
                 further_repeated.append(notice)
 
-    print("Further defaulters count: {}".format(len(further_defaulters)))
-    print("Further repeated count: {}".format(len(further_repeated)))
+    print(("Further defaulters count: {}".format(len(further_defaulters))))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:104]: These ones as well

File: migration/export_database.py [hunk 8]
```diff
@@ -138,10 +128,10 @@ def start_database_export():
     new_mongodb_uri = args.new_mongodb_uri
 
     if(args.defaulter_filename):
-        print("Defaulter File: {}".format(args.defaulter_filename))
-        print("Target DB: {}".format(new_mongodb_uri))
+        print(("Defaulter File: {}".format(args.defaulter_filename)))
```
Comment (harshkhandeparkar) [inline migration/export_database.py:131]: And here

**Comment by Teerath-Agarwal** [comment]
Thanks @harshkhandeparkar for pointing it out. I'll remove it. In the earlier version, it prints those brackets if executed using python 2. Even I was wondering that they were not required.

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by chirag-ghosh** [comment]
Thanks @Teerath-Agarwal for this PR and @harshkhandeparkar for reviewing. Merging this.

**Pull request merged**